content/F2015/F15-event-clearpath-tech-talk.md

```diff
@@ -0,0 +1,22 @@
+Title: Clearpath Tech Talk
+Date: 2015-11-10 18:00
+Category: Events
+Tags: industry, talks
+Slug: clearpath-tech-talk
+Author: Evy Kassirer
+Summary: Join us for a talk by female engineers from Clearpath Robotics! 
+
+They will be discussing the challenges of building autonomous software for 
```

Comment:
Suggest you repeat the line in the summary here in the body. Otherwise the opening "They" has no context.